webknossos/webknossos/geometry/vec3_int.py

```diff
@@ -72,6 +72,12 @@ def from_vec_or_int(vec_or_int: Union["Vec3IntLike", int]) -> "Vec3Int":
         else:
             return Vec3Int(vec_or_int)
 
+    @staticmethod
+    def from_str(string: str):
+        s = eval(string)
```

Comment:
Please don't use `eval`. That is a security vulnerability because it can execute arbitrary code.

Instead you could use `tuple(int(a.strip()) for a in string.split(","))`